GeneralsMD/Code/Libraries/Source/WWVegas/WWDebug/wwprofile.cpp

```diff
@@ -72,7 +76,11 @@ static unsigned ProfileStringCount;
 
 unsigned WWProfile_Get_System_Time()
 {
+#ifdef _WIN32
 	return TIMEGETTIME();
+#else
+	return 0;
```

Comment:
Use clock here as well?
Personally, I'd prefer to use "GetTickCount" in both cases, to use the high-precision timer.
Is there any reason against that?